tests/end_to_end/netsh_test.cpp

```diff
@@ -1,146 +1,14 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
-#include <windows.h>
-#include <netsh.h> // Must be included after windows.h
-#include <string.h>
-#include "bpf.h"
-#include "capture_helper.hpp"
-#include "catch_wrapper.hpp"
-#include "elf.h"
 #pragma warning(push)
 #pragma warning(disable : 4200)
 #include "libbpf.h"
 #pragma warning(pop)
-#include "links.h"
-#include "maps.h"
-#include "pins.h"
+#include "netsh_test_helper.h" // Has to be above links.h
```

Comment:
The comment implies we have a bug.  Every header should internally include any prerequisites.